js_modules/dagster-ui/packages/ui-core/src/app/AppProvider.tsx

```diff
@@ -109,13 +111,19 @@ export const AppProvider = (props: AppProviderProps) => {
   const retryLink = React.useMemo(() => {
     return new RetryLink({
       attempts: {
-        max: 2,
-        retryIf: (error, _operation) => {
-          return error && error.statusCode && [502, 503, 504].includes(error.statusCode);
+        max: 3,
+        retryIf: async (error, _operation) => {
```

Comment:
`_operation` unused

Reply:
I'd be ok with rm'ing these or leaving them with the `_` since we'd have to dig into the types to see what params are available on this RetryLink class